src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -2575,10 +2651,12 @@ import Color from 'color';
       });
 
       refresh_overlay.on('mouseover', () => {
```

Comment:
These can probably be abstracted as reusable callbacks (for 'mouseover' and 'mouseout'), later on.

e.g. `refresh.on('mouseover', mouseOverCallback)`;

Reply:
this is valid feedback but in general I have concerns about improving the code. is that `mouseoverCallback` function going to have the correct concept of `this`...?

Reply:
You're right, worthy consideration. Guess this can be studied later.